test/test_v2/test_common.c

```diff
@@ -184,6 +184,8 @@ void parse_cmd(int argc, char **argv, test_params *params, validation_params *v_
             fprintf(stderr, "\t-t <>    set timeout\n");
             fprintf(stderr, "\t-o out   redirect clients logs to file out.<rank>\n");
             fprintf(stderr, "\t-d str   assign ranks to servers, for example: 0:0,1:1:2,3,4\n");
+            fprintf(stderr, "\t-m num   fence time multiplier (similar to an error bar)\n");
```

Comment:
This is misleading.
these options are specific to the fence test only.
We need to allow tests to pass the callback here that can parse test-specific items and append corresponding options to the --help output

Reply:
Also need an option to enable fence timing.
I suggest having it disabled by default as it requires a quiet system for stable results.

Reply:
so the launch cmd should be something like:
```
./pmix_test -s 4 -n 16 -d '0:0,1,3,5,7,9;1:2,4,6,8;2:10,12;3:11,13,14,15' -- ./$test <test args>
```
The pmix_test should take whatever comes after the "-" symbol (or whatever separator that is convenient) and treat it as an exec command. And it needs to append to it the parameters option provided by the pmix_test itself,

Reply:
should work according to your specification now.